src/igor/sub.go

```diff
@@ -54,11 +55,12 @@ var subK string       // -k flag
 var subI string       // -i
 var subN int          // -n
 var subC string       // -c
-var subT int          // -t
+var subT string       // -t
 var subS bool         // -s
 var subA string       // -a
 var subW string       // -w
 var subProfile string // -profile
+var duration int      // parsed subT Duration in minutes
```

Comment:
Does this need to be a global? Looks like it's passed around as an argument.

Reply:
Yeah, wow. Not sure what my thought process there was. Fixed.